ci/circle

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python2.7
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import os
+import sys
+from math import ceil
+from subprocess import call
+
+BOLD = '\033[1m'
+RED = '\033[91m'
+GREEN = '\033[92m'
+YELLOW = '\033[33m'
+RESET = '\033[0m'
+
+NUM_WORKERS = 4
```

Comment:
CIRCLE_NODE_TOTAL
is the total number of parallel VMs being used to run your tests on each push.

https://circleci.com/docs/parallel-manual-setup#env-splitting